ydb/core/base/events.h

```diff
@@ -172,6 +172,7 @@ struct TKikimrEvents : TEvents {
         ES_PQ_PARTITION_CHOOSER,
         ES_GRAPH,
         ES_REPLICATION_SERVICE,
+        ES_CHANGE_EXCHANGE,
```

Comment:
Maybe it's better to swap ES_CHANGE_EXCHANGE and ES_CHANGE_EXCHANGE_DATASHARD? If ES_CHANGE_EXCHANGE were used somewhere moving it would cause hidden issues.